docs/bundle/multiple_queues.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+parent: "Symfony bundle"
+title: Multiple queues
+nav_order: 13
+---
+{% include support.md %}
+
+# Multiple queues
+
+One of the most powerful features of this bundle is the ability to run multiple queues with the same transport/client. 
+
+## Different queues for topic routing and processing
+
+In the default configuration both the topic routing and the job processing tasks are written to the `default` queue. If you want to separate those two types of tasks you'll have to change the [configuration](config_reference.md) in `config/packages/enqueue.yaml` like so:
+
+```yaml
+enqueue:
+    default:
+        transport:            # Required
+        client:
+            traceable_producer:   true
+            prefix:               enqueue
+            separator:            .
+            app_name:             app
+            router_topic:         default
+            router_queue:         router # default is: default
+            router_processor:     null
+            redelivered_delay_time: 0
+            default_queue:        default
+```
+
+## Multiple topic queues
+
+Configuration for the different queues happens on the [processor](message_processor.md)/subscriber level - the queue is either defined in `config/services.yml` like so:
+
+```yaml
+# src/AppBundle/Resources/services.yml
+
+services:
+  AppBundle\Async\SayHelloProcessor:
+    tags:
+        # registers as topic processor
+        - { name: 'enqueue.processor', topic: 'hello', queue: 'say' }
+```
+
+or in the processor itself like so:
+
+```php
+<?php
+use Enqueue\Client\CommandSubscriberInterface;
+use Interop\Queue\Processor;
+
+class SayHelloProcessor implements Processor, TopicSubscriberInterface
```

Comment:
fyi: you don't have to implement TopicSubscriberInterface interface if you use  `enqueue.processor` tag.

Reply:
If you implement `TopicSubscriberInterface` then a tag is not needed as the service is autoconfigured with `enqueue.topic_subscriber` tag.